services/Mumble/MumbleUser.ts

```diff
@@ -36,6 +37,55 @@ export class MumbleUserService extends MumbleService {
     return responseApi as IUserMumble;
   }
 
+  async getAllFollowers({
+    token,
+    identifier,
+    followers,
+    url,
+  }: {
+    token: string;
+    identifier: string;
+    followers?: string[];
+    url?: string;
+  }): Promise<string[]> {
+    const followersIntern: string[] = followers ? followers : [];
+    const queryParams = new URLSearchParams({ limit: '100', offset: '0' });
+    const urlIntern = url
+      ? url
+      : EEndpointsBackend.USER_FOLLOWERS.replace('*identifier*', identifier) +
+        `?${queryParams.toString()}`;
+    const responseApi = (await this.performRequest({
+      method: EApiMethods.GET,
+      path: urlIntern,
+      token,
+      message: 'Getting user followers',
+      headers: {
+        Authorization: `Bearer ${token}`,
+        accept: 'application/json',
+      },
+    })) as IMumbleFollowersApiResponse;
+
+    responseApi.data.map((follower) => {
+      followersIntern.push(follower.id);
+    });
+
+    if (responseApi.next) {
+      await this.getAllFollowers({
+        token,
+        identifier,
+        url: responseApi.next,
+        followers: followersIntern,
+      });
+    }
+
+    // rlu cache can be something here. we avoid to make 20 requests (or more) to get all followers
```

Comment:
I created an issue for deeper investigation: https://github.com/ost-cas-fee-adv-23-24/nextjs-app-football-fanatics/issues/31